runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -186,8 +186,12 @@ private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 			StringBuilder stringBuilder = new StringBuilder(" ");
 			for (KitType kitType : KitType.values())
 			{
-				ItemDefinition itemComposition =
-					itemManager.getItemDefinition(actor.getPlayerAppearance().getEquipmentId(kitType));
+				if (kitType.equals(KitType.RING) || kitType.equals(KitType.AMMUNITION))
```

Comment:
`KitType` is an enum so you can use `==`

Reply:
Changed